src/client/pages/App.tsx

```diff
@@ -2,20 +2,37 @@ import './App.css'
 import { Route, Routes } from 'react-router-dom'
 import { Layout } from './Layout'
 import Content from '../components/layouts/Content/Content'
-import New from '../components/layouts/GamePage/NewPage'
+import NewPage from '../components/layouts/GamePage/NewPage'
 import Ongoing from '../components/layouts/GamePage/Ongoing'
 import EndPage from '../components/layouts/GamePage/EndPage'
 import DetailPage from '../components/layouts/GamePage/DetailPage'
+import { useContext } from 'react'
+import { GameContext } from '../../api/GameContext'
+import { GameState } from '../../api/GameState'
 
 function App() {
+  const { state } = useContext(GameContext)
+
+  const renderContent = () => {
+    switch (state.gameState) {
+      case GameState.NOT_STARTED:
+        return <NewPage />
+      case GameState.START:
+      case GameState.ONGOING:
+        return <Ongoing />
+      case GameState.FINISHED:
+        return <EndPage />
+      default:
+        return <NewPage />
```

Comment:
Default wäre für mich das NOT_STARTED, da dies auch der default state sein sollte. Ausserdem sind START und ONGOING das gleiche, also ein State zu viel (schon vorher bemerkt)

Reply:
done in feature/limit-requests